project/src/main/java/com/google/sps/servlets/UserProfileUpdateServlet.java

```diff
@@ -37,6 +37,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
               .add("eventsHosting", CommonUtils.createJsonArray(new HashSet<>()))
               .add("eventsParticipating", CommonUtils.createJsonArray(new HashSet<>()))
               .add("eventsVolunteering", CommonUtils.createJsonArray(new HashSet<>()))
+              .add("imageUrl", "")
```

Comment:
Here, it might be good to add constants for imageUrl, eventsHosting etc.

Reply:
Good idea, done